Xamarin.Forms.Platform.Tizen/Renderers/EntryRenderer.cs

```diff
@@ -21,7 +21,12 @@ public EntryRenderer()
 			RegisterPropertyHandler(Entry.ReturnTypeProperty, UpdateReturnType);
 			RegisterPropertyHandler(InputView.IsSpellCheckEnabledProperty, UpdateIsSpellCheckEnabled);
 			RegisterPropertyHandler(Entry.IsTextPredictionEnabledProperty, UpdateIsSpellCheckEnabled);
-			RegisterPropertyHandler(Specific.FontWeightProperty, UpdateFontWeight);
+			RegisterPropertyHandler(Entry.IsReadOnlyProperty, UpdateIsReadOnly);
+
+			if (TizenPlatformServices.AppDomain.IsTizenSpecificAvailable)
```

Comment:
Is this from a merge?

Reply:
This is mistake in process of rebasing ;(

Reply:
I will fix this in a couple of days :)